aula-css/exercicios/photo-grid/aula-1/detstyle.css

```diff
@@ -0,0 +1,93 @@
+*{
+  font-family: 'Kumbh Sans', sans-serif;;
+}
+body{
+  background-color: #eaeaea;
+}
+
+.container{
+  display: flex;
+  align-items: center;
+  justify-content: center; 
+  max-width: 1000px;
+  margin: 0 auto ;
+  flex-wrap: wrap;
+}
+a:link, a:visited {
+  text-decoration: none;
+}
+a:link {
+  color: black;
+}
+a:visited {
+  color: gray;
+}
+.header-text a{
+  font-weight: bold;
+}
+.header{
+  margin-bottom: 20px;
+  margin-top: 20px;
+}
+.header-text{
+  
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+  max-width: 1000px;
+  margin: 0 auto ;
+  flex-wrap: wrap;
+}
+
+.header-title{
+  margin: 0 auto;
+}
+
+.card{
+  max-width: 1000px;
+  background-color: white;
+  margin: 0 10px 10px 0;
+  padding: 0;
+  display: flex;
+  flex-wrap: wrap;
+  border: #999999 1px solid;
+}
+.card-text{
+  width: 100%;
+  
+}
+.card-text-title{
+  padding: 10px;
+}
+.card-text-description{
+  padding: 30px;
+}
+.card hr {
+  margin: 0;
+}
+.card-img{
+  margin: 0;
+  padding: 0;
+  width: 100%;
+  
+}
+.img-gallery{
+  width: 100%;
+  height: 100%;
+}
+.pic-profile{
+  max-height: 50px;
+  float: left;
```

Comment:
Se for o que estou pensando, você conseguiria resolver com flexbox utilizando o `display: flex` e o `justify-content: space-around` para jogar a imagem de um lado e o texto de outro